proxy/api/src/http/error.rs

```diff
@@ -81,7 +84,7 @@ pub async fn recover(err: Rejection) -> Result<impl Reply, Infallible> {
             )
         } else if let Some(err) = err.find::<Routing>() {
             match err {
-                Routing::MissingOwner => {
+                Routing::MissingOwner | Routing::SealedKeystore => {
```

Comment:
We might benefit from separating them. The sealed keystore could be a `403` and the missing owner a `401`.